lib/livebook/hubs/team_client.ex

```diff
@@ -382,6 +416,7 @@ defmodule Livebook.Hubs.TeamClient do
   end
 
   defp handle_event(:deployment_group_updated, %Teams.DeploymentGroup{} = deployment_group, state) do
+    deployment_group = deploy_apps(state.deployment_group_id, deployment_group, state.derived_key)
```

Comment:
I'm not sure we should be redeploying on any change? @josevalim

Reply:
We are going to change those events  in a future PR. We will be receiving explicit events for apps being deployed and undeployed, so don't worry here.